crypto/bio/bio_lib.c

```diff
@@ -778,9 +778,9 @@ void bio_cleanup(void)
 {
 #ifndef OPENSSL_NO_SOCK
     bio_sock_cleanup_int();
-    CRYPTO_THREAD_lock_free(bio_lookup_lock);
+    /* Clear the local handle to the global lock, freed elsewhere. */
     bio_lookup_lock = NULL;
 #endif
-    CRYPTO_THREAD_lock_free(bio_type_lock);
+    /* Clear the local handle to the global lock, freed elsewhere. */
     bio_type_lock = NULL;
```

Comment:
Do you see the typo here? I mean why was it two calls to CRYPTO_THREAD_lock_free [and two commentary lines]?

Reply:
I'm afraid I don't see the typo, no.  There were two different locks getting freed, and I put int two comments because a reader might assume that the first one is bounded by the scope of the preprocessor conditional.

Reply:
Oh! Brainslip! I apologize for diversion.